### PR TITLE
Fix for issue #692 

### LINKED ISSUE #692: Prerequisites are missing for Setting up OpenEBS with Kubernetes on Google Container Engine
BUG REPORT 

If this is a BUG REPORT, please:
Setting up OpenEBS with Kubernetes on Google Container Engine. Instead, it is mentioned as " Minimum requirements for container cluster are as follows:"

### PR DESCRIPTION
**Pull request by ksatchit**
**What this PR does / why we need it**:

Makes updates to the openebs-on-premise-deployment-using-ansible guide. We are adding the 
minimum system requirements 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #692

### CODE REVIEW

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 1]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
 
 - At least 3 linux machines of above type, i.e., either VMs or bare-metal, if deploying the setup in a hyperconverged mode
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: bare-metal = bare metal

Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: hyperconverged = hyper converged

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 2]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
 
 - At least 3 linux machines of above type, i.e., either VMs or bare-metal, if deploying the setup in a hyperconverged mode 
   (with K8s as well as OpenEBS residing on the same machines) OR 5 linux machines (with K8s and OpenEBS running on separate machines)
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: machines = machine

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 3]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
...
-  
+
 - The above instruction assumes a minimal setup with a test-harness, K8s/OpenEBS master and a single K8s minion/OpenEBS node.
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: minion = node

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 4]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
 
 - At least 3 linux machines of above type, i.e., either VMs or bare-metal, if deploying the setup in a hyperconverged mode 
   (with K8s as well as OpenEBS residing on the same machines) OR 5 linux machines (with K8s and OpenEBS running on separate machines)
-  
+
 - The above instruction assumes a minimal setup with a test-harness, K8s/OpenEBS master and a single K8s minion/OpenEBS node. 
   The masters and nodes can be scaled if the user so desires
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: so desires = requires.

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 5]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
...
 
+- The minimum system requirements for the above machines are :
+  
+  - 10GB hard disk space
+  - 1GB RAM
+  - 1 core processor
+
+  Note: The min requirements are based on the scenario where the VMs are built using the vagrantfile in the ansible
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: min = minimum

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 6]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
...
+  repo.
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: repo = repository

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 7]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
...
+- The minimum system requirements for the above machines are :
+  
+  - 10GB hard disk space
+  - 1GB RAM
+  - 1 core processor
+
+  Note: The min requirements are based on the scenario where the VMs are built using the vagrantfile in the ansible
+  repo.  
+
 - All linux machines are required to have :
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: are required to have = require

File: e2e/ansible/openebs-on-premise-deployment-guide.md [hunk 8]
```diff
@@ -24,10 +24,19 @@ The following instructions have been verified on:
...
+  - 10GB hard disk space
+  - 1GB RAM
+  - 1 core processor
+
+  Note: The min requirements are based on the scenario where the VMs are built using the vagrantfile in the ansible
+  repo.  
+
 - All linux machines are required to have : 
   
   - Basic development packages (dpkg-dev,gcc,g++,libc6-dev,make,libssl-dev,sshpass,curl)
```
Comment (mahebbar) [inline e2e/ansible/openebs-on-premise-deployment-guide.md]: space after each comma

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 1 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: @mahebbar , this is a standard usage/used as a single node

Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 2 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 3 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 4 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 5 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

File: e2e/ansible/openebs-on-premise-deployment-guide.md (hunk 8 above)
Comment (ksatchit) [reply e2e/ansible/openebs-on-premise-deployment-guide.md]: done

**Pull request merged**